catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -273,21 +353,21 @@ private Query getQueryRequest(String cql) throws CQLException {
     public void testExecuteQueryCacheEnabledWithSingleSource() throws Exception {
         Set<String> srcIds = new HashSet<>(1);
         srcIds.add("id");
-        List<String> modes = cacheQuery(srcIds, 2);
+        List<String> modes = cacheQuery(srcIds, 3);
 
-        assertThat(modes.size(), is(2));
-        assertThat(modes, hasItems("cache", "update"));
+        assertThat(modes.size(), is(3));
+        assertThat(modes, hasItems("cache", "cache", "update"));
     }
 
     @Test
     public void testExecuteQueryCacheEnabledWithMultipleSources() throws Exception {
         Set<String> srcIds = new HashSet<>(2);
         srcIds.add("id1");
         srcIds.add("id2");
-        List<String> modes = cacheQuery(srcIds, 3);
+        List<String> modes = cacheQuery(srcIds, 4);
 
-        assertThat(modes.size(), is(3));
-        assertThat(modes, hasItems("cache", "update", "update"));
+        assertThat(modes.size(), is(4));
```

Comment:
✏️  It would help readability to extract the value 4 and set as a final constant at the top of the class. That would also help with your cacheQuery() call readability.